docs/hacking_howto.md

```diff
@@ -120,71 +129,36 @@ running this command::
 
 ### Get AAQ working
 
-1. Add a product with a slug matching one of the product values in `kitsune/questions/config.py`.
-You can do this through the admin interface at `/admin/products/product/add/`.
-
-    For instance, with a `config.py` value like:
-
-    ```python
-    ...
-    "name": _lazy("Firefox"),
-    "product": "firefox",
-    ...
-    ```
-
-    Create a product in the admin interface with its `slug` set to `firefox`.
-
-2. Add a topic matching a category from that product config,
-and associate it with the product you just created.
-You can do this through the admin interface at `/admin/products/topic/add/`.
-
-    For instance, with a category with a `config.py` value like:
-
-    ```python
-    ...
-    "topic": "download-and-install",
-    ...
-    ```
+1.  Add a product with a slug matching one of the product values in `kitsune/questions/config.py`.
+    You can do this through the admin interface at `/admin/products/product/add/`.
 
-    Create a topic in the admin interface with its `slug` set to `download-and-install` and its product set to the product you just created.
+        For instance, with a `config.py` value like:
 
-3. Finally add an AAQ locale for that product.
-You can do this through the admin interface at `/admin/questions/questionlocale/add/`.
+        ```python
+        ...
+        "name": _lazy("Firefox"),
+        "product": "firefox",
+        ...
+        ```
 
+        Create a product in the admin interface with its `slug` set to `firefox`.
 
-### Get search working
-
-First, make sure you have run the "Install Sample Data" step above,
-or have entered data yourself through the admin interface.
-
-1. Enter into the web container
-    ```
-    docker-compose exec web bash
-    ```
+2.  Add a topic matching a category from that product config,
+    and associate it with the product you just created.
+    You can do this through the admin interface at `/admin/products/topic/add/`.
 
-2. Build the indicies
-    ```
-    $ ./manage.py esreindex
```

Comment:
This is the only bit no longer relevant, I'd suggest updating it to be the es7 command - both the nunjucks precompile and analytics.js note are still very relevant.